🚑 [Hotfix] 프로세스 시작시에 BeanDefinitionStoreException 발생

## ❗ 버그 설명
프로세스 시작시에 BeanDefinitionStoreException 발생

## 🤔 기대한 동작
- 프로세스 정상 실행

## 🐞 실제 동작
- 프로세스 시작시에 BeanDefinitionStoreException 발생
org.springframework.beans.factory.BeanDefinitionStoreException: Failed to parse configuration class [verbly.spring.Application]
	at org.springframework.context.annotation.ConfigurationClassParser.parse(ConfigurationClassParser.java:186) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.processConfigBeanDefinitions(ConfigurationClassPostProcessor.java:417) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ConfigurationClassPostProcessor.postProcessBeanDefinitionRegistry(ConfigurationClassPostProcessor.java:290) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanDefinitionRegistryPostProcessors(PostProcessorRegistrationDelegate.java:349) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.support.PostProcessorRegistrationDelegate.invokeBeanFactoryPostProcessors(PostProcessorRegistrationDelegate.java:118) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.support.AbstractApplicationContext.invokeBeanFactoryPostProcessors(AbstractApplicationContext.java:789) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.support.AbstractApplicationContext.refresh(AbstractApplicationContext.java:607) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.boot.web.servlet.context.ServletWebServerApplicationContext.refresh(ServletWebServerApplicationContext.java:146) ~[spring-boot-3.3.10.jar:3.3.10]
	at org.springframework.boot.SpringApplication.refresh(SpringApplication.java:754) ~[spring-boot-3.3.10.jar:3.3.10]
	at org.springframework.boot.SpringApplication.refreshContext(SpringApplication.java:456) ~[spring-boot-3.3.10.jar:3.3.10]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:335) ~[spring-boot-3.3.10.jar:3.3.10]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1363) ~[spring-boot-3.3.10.jar:3.3.10]
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:1352) ~[spring-boot-3.3.10.jar:3.3.10]
	at verbly.spring.Application.main(Application.java:14) ~[main/:na]
Caused by: org.springframework.context.annotation.ConflictingBeanDefinitionException: Annotation-specified bean name 'reviewController' for bean class [verbly.spring.domain.review.controller.ReviewController] conflicts with existing, non-compatible bean definition of same name and class [verbly.spring.domain.quizreview.controller.ReviewController]
	at org.springframework.context.annotation.ClassPathBeanDefinitionScanner.checkCandidate(ClassPathBeanDefinitionScanner.java:361) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ClassPathBeanDefinitionScanner.doScan(ClassPathBeanDefinitionScanner.java:288) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ComponentScanAnnotationParser.parse(ComponentScanAnnotationParser.java:128) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ConfigurationClassParser.doProcessConfigurationClass(ConfigurationClassParser.java:319) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ConfigurationClassParser.processConfigurationClass(ConfigurationClassParser.java:259) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ConfigurationClassParser.parse(ConfigurationClassParser.java:195) ~[spring-context-6.1.18.jar:6.1.18]
	at org.springframework.context.annotation.ConfigurationClassParser.parse(ConfigurationClassParser.java:165) ~[spring-context-6.1.18.jar:6.1.18]
	... 13 common frames omitted
## 🖥 환경 정보
- Spring Boot: 3.3.10
- JDK: 17
- DB: mysql
- OS: window

## 📸 에러 로그 or 캡처 (선택)

--- a/src/main/java/verbly/spring/domain/quizreview/controller/ReviewRestController.java
+++ b/src/main/java/verbly/spring/domain/quizreview/controller/ReviewRestController.java
@@ -23,7 +23,7 @@ import verbly.spring.global.security.utils.SecurityUtils;
 @RestController
 @RequiredArgsConstructor
 @RequestMapping("/api/quizzes")
-public class ReviewController {
+public class ReviewRestController {
 
     private final ReviewCommandService reviewCommandService;
     private final ReviewQueryService reviewQueryService;

